lib/collection.js

```diff
@@ -277,12 +277,13 @@ const DEPRECATED_FIND_OPTIONS = ['maxScan', 'fields', 'snapshot'];
  * @param {number} [options.maxScan] DEPRECATED: Limit the number of items to scan.
  * @param {number} [options.min] Set index bounds.
  * @param {number} [options.max] Set index bounds.
- * @param {boolean} [options.showDiskLoc] Show disk location of results.
+ * @param {boolean} [options.showDiskLoc=false] DEPRECATED. Show disk location of results.
```

Comment:
Should this be defaulting to false?

Reply:
It shouldn't be. I left it as `false` because it is deprecated in favor of `showRecordId`. I can remove the default if you think that would be more clear!